src/components/chatX/ChatXPage.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Link } from 'react-router';
+import checkAuth from '../requireAuth';
+import ChatXBody from './ChatXBody';
+import ChatXInput from './ChatXInput';
+import { messageSent, getInitial10Messages, getNewMessages, getMessageListStartAtKey } from '../../actions/chatXActions';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+export class ChatXPage extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+        this.user = this.props.user.email;
+
+        this.state = {
+            messageList: [],
+            newMessage: ""
+        };
+
+        this.newMsgCallback = this.newMsgCallback.bind(this);
+        this.getInitialMessageList = this.getInitialMessageList.bind(this);
+        this.updateInputState = this.updateInputState.bind(this);
+        this.sendMessage = this.sendMessage.bind(this);
+        this.setScrollToBottom = this.setScrollToBottom.bind(this);
+    }
+
+    componentDidMount() {
+        this.props.actions.getNewMessages(this.newMsgCallback);
```

Comment:
Yes, that's where to bind the listener via `getNewMessages`.. but less sure about passing the callback `this.newMsgCallback` here :)